project_euler/problem_04/sol2.py

```diff
@@ -10,24 +10,22 @@
 
 def solution(n: int = 998001) -> int:
     """Returns the largest palindrome made from the product of two 3-digit
-    numbers which is less than n or -1 if no such product exists.
+    numbers which is less than n.
 
     >>> solution(20000)
     19591
     >>> solution(30000)
     29992
     >>> solution(40000)
     39893
-    >>> solution(10000)
-    -1
     """
     answer = 0
     for i in range(999, 99, -1):  # 3 digit numbers range from 999 down to 100
         for j in range(999, 99, -1):
-            product_string = str(i * j)
-            if product_string == product_string[::-1] and i * j < n:
+            t = str(i * j)
```

Comment:
Expanding variable names is fine.

Reply:
Gotcha, I'll add the expanded variable names shortly.